games/f.yaml

```
@@ -326,11 +326,12 @@
   lang: C
   license:
   - GPL2
-  development: active
+  development: very active
```

```suggestion
  development: active
```
Last commit 24 days ago; not very active

---

Done